hawtio-util/src/test/java/io/hawt/util/FilesTest.java

```diff
@@ -0,0 +1,298 @@
+    PowerMockito.when(file.isDirectory()).thenReturn(true);
+    Assert.assertEquals("application/zip", Files.getMimeType(file));
+
+    PowerMockito.when(file.isDirectory()).thenReturn(false);
+    
+    PowerMockito.when(file.getName()).thenReturn(".xml");
+    Assert.assertEquals("application/xml", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".wadl");
+    Assert.assertEquals("application/wadl+xml", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".wsdl");
+    Assert.assertEquals("application/wsdl+xml", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".xsd");
+    Assert.assertEquals("application/xsd+xml", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".json");
+    Assert.assertEquals("application/json", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".html");
+    Assert.assertEquals("application/html", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".htm");
+    Assert.assertEquals("application/html", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".properties");
+    Assert.assertEquals("text/x-java-properties", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".jpg");
+    Assert.assertEquals("image/jpeg", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".jpeg");
+    Assert.assertEquals("image/jpeg", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".png");
+    Assert.assertEquals("image/png", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".gif");
+    Assert.assertEquals("image/gif", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".svg");
+    Assert.assertEquals("image/svg+xml", Files.getMimeType(file));
+
+    PowerMockito.when(file.getName()).thenReturn(".txt");
+    Assert.assertEquals("text/plain", Files.getMimeType(file));
+  }
+
+  @Test
+  public void assertExists() throws Exception {
+    // Setup mocks
+    final File file = PowerMockito.mock(File.class);
+
+    // Arrange, Act & Assert result
+    PowerMockito.when(file.exists()).thenReturn(true);
+    Files.assertExists(file);
+    // Method returns void, testing that no exception is thrown
+
+    PowerMockito.when(file.exists()).thenReturn(false);
+    thrown.expect(IllegalArgumentException.class);
+    Files.assertExists(file);
+    // Method is not expected to return due to exception thrown
+  }
+
+  @Test
+  public void assertFileExists() throws Exception {
+    // Setup mocks
+    final File file = PowerMockito.mock(File.class);
+
+    // Arrange, Act & Assert result
+    PowerMockito.when(file.exists()).thenReturn(true);
+    PowerMockito.when(file.isFile()).thenReturn(true);
+    Files.assertFileExists(file);
+    // Method returns void, testing that no exception is thrown
```

Comment:
`// Method returns void, testing that no exception is thrown`

`// Method is not expected to return due to exception thrown`

:thinking: